_data/hosting_providers.json

```
@@ -1,4 +1,14 @@
 [
+    {
```

*extremely* small nit, but the spacing of this open brace is off by one.
```suggestion
   {
```